test/clj/rems/test_browser.clj

```diff
@@ -162,8 +171,10 @@
   Optionally give `:index` when several items match. It starts from 1."
   [label text & [opts]]
   (assert (> (:index opts 1) 0) "indexing starts at 1") ; xpath uses 1, let's keep the convention though we don't use xpath here because it will likely not work
+  (assert (> (:form-nth opts 1) 0) "indexing starts at 1") ; css pseudo-selector works the same way as above
 
-  (let [el (nth (btu/query-all [{:css ".fields"}
+  (let [el (nth (btu/query-all [{:css (str ".application-fields:nth-child(" (or (:form-nth opts) 1) ")")}
```

Comment:
for some reason xpath selector (`:index`) did not work for me, so i resorted to using css pseudo-selector

Reply:
Did you account for that xpath indexing starts at `1`? There are some helpers/old code that has had to handle some of these.

Reply:
Ah, now I also see that this used to be `.fields` here.